jssource/src_files/include/javascript/EmailsComposeViewModal.js

```diff
@@ -114,10 +114,10 @@
     self.emailComposeView = null;
```

Comment:
After testing your change I noticed tinymce editor is broken. We need to fix this as well as fixing the issue in the cases module.

![screenshot_20170628_113353](https://user-images.githubusercontent.com/12231216/27633871-d84090be-5bf8-11e7-9e67-a8cc24807078.png)